src/ForkCMS/Utility/Geolocation.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace ForkCMS\Utility;
+
+use Backend\Core\Language\Language;
+use Common\ModulesSettings;
+use JeroenDesloovere\Geolocation\Geolocation as API;
+use JeroenDesloovere\Geolocation\Result\Coordinates;
+use JeroenDesloovere\Geolocation\Exception;
+use Symfony\Component\Intl\Intl;
+
+class Geolocation
+{
+    /** @var API */
+    private $api;
+
+    public function __construct(ModulesSettings $settings)
+    {
+        $this->api = new API($settings->get('Core', 'google_maps_key'));
+    }
+
+    /**
+     * @param string|null $street
+     * @param string|null $streetNumber
+     * @param string|null $city
+     * @param string|null $zip
+     * @param string|null $country
+     * @return array - Example: ['latitude' => 50.8864, 'longitude' => 3.42928]
+     * @throws Exception
+     */
+    public function getCoordinates(
+        string $street = null,
+        string $streetNumber = null,
+        string $city = null,
+        string $zip = null,
+        string $country = null
+    ): array {
+        if (!empty($country)) {
+            $country = Intl::getRegionBundle()->getCountryName($country, Language::getInterfaceLanguage());
+        }
+
+        /** @var Coordinates $coordinates */
+        $coordinates = $this->api->getCoordinates(
+            $street,
+            $streetNumber,
+            $city,
+            $zip,
+            $country
+        );
+
+        return [
```

Comment:
What if the address can't be decoded?

Reply:
If any exception occurs in the vendor this will be thrown.
We just need to decide how we will handle this in Fork... 

Possible solution:
What about a try/catch around `$this->api->getCoordinates(` and return null values 
```
        return [
            'latitude' => null,
            'longitude' => null,
        ];
```
Any other suggestions?

Reply:
... using null is exactly the same what we use now in https://github.com/forkcms/forkcms/pull/2525/files#diff-3d41d594577d7d3f3003016b06d01cd9L144

I'm going to update this PR so it matches the same behavior.

Reply:
Done. @tijsverkoyen can you review https://github.com/forkcms/forkcms/pull/2525/commits/60a0d41b90d135df5f2ec07f7b0cf93510c83bae